rpm/libzypp.spec

```diff
@@ -94,7 +94,6 @@ export CXXFLAGS="%{optflags}"
       -DCMAKE_BUILD_TYPE=Release \
       -DCMAKE_SKIP_RPATH=1 \
       -DUSE_TRANSLATION_SET=${TRANSLATION_SET:-zypp} \
-      -DDISABLE_LIBPROXY=TRUE \
```

Comment:
If this would be =FALSE would it enforce this always and fail in case someone removes libproxy?

Reply:
Not it doesn't:
https://github.com/sailfishos-mirror/libzypp/blob/43bd2bf563be01d429e7930019da7682ef430c1c/CMakeLists.txt#L242

Reply:
Unfortunately it doesn't work that way